aws_lambda_builders/__main__.py

```diff
@@ -46,6 +49,22 @@ def _error_response(request_id, http_status_code, message):
     })
 
 
+def _parse_version(version_string):
+
+    if VERSION_REGEX.match(version_string):
+        return float(version_string)
```

Comment:
didn't realize you could do this :) Hope it is safe

Reply:
should be safe, we do a regex match before we get here.